s:1320:"<b>Failed tests</b><br />Test 1: Teste 1
<br />Test 2: Teste 2
<br />Test 3: Teste 3
<br /><b>Test 1: Teste 1 </b><br />Incorrect program result
<br /> --- Input ---
<br /><pre><i> 2
4
3
2
</i></pre>
<br /> --- Program output ---
<br /><pre><i>digite o nome de T1:digite o nome de T2:digite o nome de T3:digite o nome de T4:8
</i></pre>
<br /> --- Expected output (numbers)---
<br /><pre><i>8
</i></pre>
<br /><b>Test 2: Teste 2 </b><br />Incorrect program result
<br /> --- Input ---
<br /><pre><i> 6
6
6
6
</i></pre>
<br /> --- Program output ---
<br /><pre><i>digite o nome de T1:digite o nome de T2:digite o nome de T3:digite o nome de T4:21
</i></pre>
<br /> --- Expected output (numbers)---
<br /><pre><i>21
</i></pre>
<br /><b>Test 3: Teste 3 </b><br />Incorrect program result
<br /> --- Input ---
<br /><pre><i> 2,2,2,2
</i></pre>
<br /> --- Program output ---
<br /><pre><i>digite o nome de T1:digite o nome de T2:Traceback (most recent call last):
  File "tomadas.py", line 7, in &lt;module&gt;
    T2=input('digite o nome de T2:')
EOFError: EOF when reading a line
</i></pre>
<br /> --- Expected output (numbers)---
<br /><pre><i>5
</i></pre>
<br /><b>Summary of tests</b><br /><pre><i>+------------------------------+
|  3 tests run/ 0 tests passed |
+------------------------------+
</i></pre>
<br /><br />";